src/lib/emile/meson.build

```diff
@@ -24,12 +24,6 @@ elif (get_option('crypto') == 'openssl')
   emile_src += files('emile_cipher_openssl.c')
 endif
 
-if not sys_windows
-  emili_lib_deps = [lz4, rg_etc, m]
-else
-  emili_lib_deps = [lz4, rg_etc]
-endif
-
```

Comment:
Here too seems as some wrong conflict resolution, did you intended this change?

Reply:
Yes. In this specific case, `m` shouldn't even be breaking things, so we should actually fix `m` instead of removing it.

Reply:
`m` shouldn't be breaking things? why exactly?

Reply:
Why would? It is the standard math library. It'll only break if we get it the wrong way.

Reply:
I don't know exactly, but we were avoiding it at windows builds, weren't we?
How can we get it right?

Reply:
Just ensure we're following this: https://mesonbuild.com/howtox.html#add-math-library-lm-portably.

Reply:
`emili_lib_dep` seems misspelled and it is not used anywhere...